WISE-Community/plate-tectonics-boundary-explorer
# feat(UX): Repopulate student explanations

## Changes
- Repopulate most recent explanation each time user revisits or retries boundary.
- Require user to change explanation text before playing boundary animation again.

Closes #6.

## Test
- Try out each boundary type multiple times and make sure your precious explanation for that boundary is repopulated each time.
- Make sure you're required to revise your explanation before pressing play.
